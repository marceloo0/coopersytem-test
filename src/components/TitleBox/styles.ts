import styled from 'styled-components/native';

export const Container = styled.View`
  padding: 15px 20px;
  flex-direction: row;
  justify-content: space-between;
`;
export const Title = styled.Text`
  font-size: 21px;
  color: #6c6c80;
  font-weight: bold;
`;
