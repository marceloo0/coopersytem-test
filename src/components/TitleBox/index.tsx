import React from 'react';

import {Container, Title} from './styles';

interface TitleProps {
  title: string;
  description?: string;
}

const TitleBox: React.FC<TitleProps> = ({title, description}) => {
  return (
    <Container>
      <Title>{title}</Title>
      <Title>{description}</Title>
    </Container>
  );
};

export default TitleBox;
