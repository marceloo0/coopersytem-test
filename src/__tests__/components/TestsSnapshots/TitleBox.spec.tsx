import React from 'react';
import renderer from 'react-test-renderer';
import TitleBox from '../../../components/TitleBox';

test('renders correctly', () => {
  const titleBox = renderer
    .create(<TitleBox title="RESGATE DO SEU JEITO" />)
    .toJSON();
  expect(titleBox).toMatchSnapshot();
});
