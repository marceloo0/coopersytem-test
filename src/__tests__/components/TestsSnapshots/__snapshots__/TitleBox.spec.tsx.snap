// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 15,
        "paddingLeft": 20,
        "paddingRight": 20,
        "paddingTop": 15,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#6c6c80",
          "fontSize": 21,
          "fontWeight": "bold",
        },
      ]
    }
  >
    RESGATE DO SEU JEITO
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "#6c6c80",
          "fontSize": 21,
          "fontWeight": "bold",
        },
      ]
    }
  />
</View>
`;
